shared/templates/static/bash/file_permissions_sshd_private_key.sh

```diff
@@ -1,3 +1,8 @@
 # platform = multi_platform_rhel
 
-chmod 0640 /etc/ssh/*_key
+#check if files exist before attempting 
+#to change them
+if ls /etc/ssh/*_key &> /dev/null
+then
+    chmod 0640 /etc/ssh/*_key
+fi
```

Comment:
As OVAL fails when no file is present, this change it remediation won't make a difference - oscap will consider it erroneous anyway, as it will perform another evaluation run and see the rule still failing. To fix the remediation, you would have to generate the key.